wfe2/wfe_test.go

```diff
@@ -1543,7 +1543,6 @@ func TestNewECDSAAccount(t *testing.T) {
 	test.Assert(t, len(*acct.Contact) >= 1, "No contact field in account")
 	test.AssertEquals(t, (*acct.Contact)[0], "mailto:person@mail.com")
 	test.AssertEquals(t, acct.Agreement, "")
-	test.AssertEquals(t, acct.InitialIP.String(), "1.1.1.1")
```

Comment:
Might be worth preserving a single test that confirms that the WFE is hardcoding this to 0.0.0.0 now.

Reply:
We can't preserve this test unless we also preserve the code that marshals the pb back into an object we can show the subscriber.